docs/fsharp/whats-new/fsharp-9.md

```diff
@@ -0,0 +1,408 @@
+---
+title: What's new in F# 9 - F# Guide
+description: Find information on the new features available in F# 9.
+ms.date: 11/12/2024
+ms.topic: whats-new
+---
+
+# What's new in F# 9
+
+F# 9 introduces a range of enhancements that make your programs safer, more resilient, and performant. This article highlights the major changes in F# 9, developed in the [F#'s open source code repository](https://github.com/dotnet/fsharp).
+
+You can download the latest .NET SDK from the [.NET downloads page](https://dotnet.microsoft.com/download).
+
+## F# language changes
```

Comment:
I have looked at the rendered preview and it would be IMO better if the individual language changes were directly availlable as links in the generated menu on top, in the "In This article" section.

But that would mean moving the individual language changes to become a '##' each, is that possible?

Reply:
I think so. We can put language changes at H2 level and keep the rest as it is.